modules/machineset-yaml-azure.adoc

```diff
@@ -151,18 +143,17 @@ $  oc -n openshift-machine-api \
 ifndef::infra[]
 <2> Specify the node label to add.
 <3> Specify the infrastructure ID, node label, and region.
-<4> Optional: Specify the machine set name to enable the use of availability sets. This setting only applies to new compute machines.
-<5> Specify the region to place machines on.
-<6> Specify the zone within your region to place machines on. Be sure that your region supports the zone that you specify.
 endif::infra[]
 ifdef::infra[]
 <2> Specify the `<infra>` node label.
 <3> Specify the infrastructure ID, `<infra>` node label, and region.
+endif::infra[]
 <4> Optional: Specify the machine set name to enable the use of availability sets. This setting only applies to new compute machines.
-<5> Specify a taint to prevent user workloads from being scheduled on infra nodes.
+<5> Specify an image that is compatible with your instance type. The Hyper-V generation V2 images created by the installer have a `-gen2` suffix, while V1 images have the same name without the suffix.
```

Comment:
```suggestion
<5> Specify an image that is compatible with your instance type. The Hyper-V generation V2 images created by the installation program have a `-gen2` suffix, while V1 images have the same name without the suffix.
```

Reply:
hm good question - @mjpytlak what's the norm for our docs on this term?

Reply:
`Installer` is all over the install doc set, but it should be `installation program`. I typically fix these type of instances as I go.

Reply:
> hm good question - @mjpytlak what's the norm for our docs on this term?

`Installer` is all over the install doc set, but it should be` installation program`. I typically fix these type of instances as I go.

Reply:
Thanks! I'll make sure this one is correct from the get-go then :slightly_smiling_face: